documentation/http_certificate_clean.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: "Puppet Server: Puppet HTTP API: Certificate Clean"
+canonical: "/puppetserver/latest/puppet-api/v3/task-detail.html"
+---
+Certificate Clean
+===============
+
+The `certificate clean` endpoint of the CA API allows the user to revoke and delete a list
+of certificates with a single request.
+
+```
+PUT /puppet-ca/v1/clean
+Content-Type: application/json
+```
+Takes a body with one required key, `certnames`, a list of certificates to clean. Each cert
```

Comment:
This first sentence seems incomplete?  Change to "It takes a body with one ...."